examples/scan.sh

```diff
@@ -39,4 +39,4 @@ mvn ${SONAR_SCANNER_DEFAULTS} scoverage:report sonar:sonar
 # Maven scala-java
 echo -e "\nScanning Maven scala-java project."
 cd $CWD/mvn/scala-java
-mvn ${SONAR_SCANNER_DEFAULTS} clean test scoverage:report jacoco:report sonar:sonar
+mvn ${SONAR_SCANNER_DEFAULTS} clean test scoverage:report jacoco:report scala:compile sonar:sonar
```

Comment:
For some strange reason after running the tests the scapegoat report was empty _(apparently scapegoat run over the Java sources or something, i really don't know that the hell happened)_.

So, re running the Scala compile phase filled the report again. 😸

Reply:
I remember trying to get scapegoat working with maven but had issues with the report getting cleaned for multi-module projects - I'm glad you got this working.